__tests__/app.test.js

```diff
@@ -50,6 +50,31 @@ describe("/api/topics", () => {
 });
 
 describe("/api/articles", () => {
+  describe("GET", () => {
+    test("200 + returns all articles", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          const articles = body.articles;
+          articles.forEach((article) => {
+            expect(article).toEqual(
+              expect.objectContaining({
+                article_id: expect.any(Number),
+                title: expect.any(String),
+                topic: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String),
+              })
+            );
+          });
+        });
+    });
+  });
+
```

Comment:
Either within this test, or in a new one, we also want to be checking the default sort and order please